ChangeLog.md

```diff
@@ -1,3 +1,8 @@
+2023-10-10 Version 6.0.0
+
+- Update Branch iOS SDK to 3.0.0
```

Comment:
Are there any notes someone might want if they don't frequent the native ios repo?